docs/releases/4.1.txt

```diff
@@ -156,6 +156,16 @@ CSRF
 
 * ...
 
+Database backends
+~~~~~~~~~~~~~~~~~
+
+* Third-party database backends can now specify the minimum required version
+  of the database using the ``DatabaseFeatures.minimum_database_version``
+  attribute which is a tuple (e.g. (10,) means "10.0"). If a minimum version
```

Comment:
I guess my suggested wording isn't quite right since "10," is displayed as "10". It would have to be "(10, 0)" to be displayed as 10.0.